apispec/openapi.json

```diff
@@ -5595,7 +5595,7 @@
       "Mw-Access": {
         "name": "Mw-Access",
         "in": "header",
-        "description": "Indicates the customer id. It is \"impersonation\" and  it’s not exposed to the customers. It is only for admin use.",
+        "description": "Indicates the customer id. It is \"impersonation\" and it is not exposed to customers. It is only for admin use.",
```

Comment:
Use "ID" instead of "id".
If the Mw-Access is only used internally, I suggest you OMIT any mention of Mw-Access from the public documentation.
In CDN360, I do not describe fields that are only for internal use.